app/views/planning_applications/show.html.erb

```diff
@@ -1,101 +1,124 @@
 <%= render "assessment_dashboard" do %>
-  <% unless @planning_application.withdrawn?  ||  @planning_application.returned?  %>
-    <ol class="app-task-list">
-      <li>
-        <h2 class="app-task-list__section">
-          <span class="app-task-list__section-number"><%= t("#{@planning_application.status}.heading") %></span>
-        </h2>
-        <ul class="app-task-list__items">
-          <li class="app-task-list__item">
+  <% unless @planning_application.withdrawn? || @planning_application.returned? %>
+    <h2 class="app-task-list__section">
+      <span class="app-task-list__section-number"><%= t("#{@planning_application.status}.heading") %></span>
```

Comment:
I don't think this was made particularly clear from the designs, but I've checked with Ali, and this heading should always be "Process Application", so we can just put this in without doing the `t()` thing.